feature-libs/checkout/occ/adapters/occ-checkout-delivery.adapter.ts

```diff
@@ -24,9 +24,39 @@ export class OccCheckoutDeliveryAdapter implements CheckoutDeliveryAdapter {
     protected converter: ConverterService
   ) {}
 
-  protected getCartEndpoint(userId: string): string {
-    const cartEndpoint = 'users/' + userId + '/carts/';
-    return this.occEndpoints.getEndpoint(cartEndpoint);
+  protected deliveryAddressesEndpoint(userId: string, cartId: string): string {
+    return this.occEndpoints.getUrl('deliveryAddresses', { userId, cartId });
+  }
+
+  protected setDeliveryAddressEndpoint(
```

Comment:
I may be missing something, but since this `setDeliveryAddressEndpoint` returns an endpoint, should it not be called `get{something}Endpoint`?